features/projects/components/project-list/error-page/error-page.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {
+  AlertIcon,
+  AlertWrapper,
+  ArrowIcon,
+  Container,
+  Message,
+  RefreshButton,
+  Wrapper,
+} from "./error-style";
```

Comment:
Instead of importing each component you can also use a wildcard like

```
import * as S from "./error-style";
```

This way you also get autocomplete when you e.g. type `S.A` you'll see all components starting with "A"

Reply:
oh yeah! I forgot about the "as" purpose lol! will update it shortly. Thanks!!